### PR TITLE
Update the icons list for the last version 5.4.1

### PR DESCRIPTION
**Pull request by leandrosimoes**
- Update the icons list for the last version 5.4.1
- Created a IconTypes object that contains the font families so you can pass through the FontAwesome component properties (Why and How: README)
- Created a parseIconName function so you can parse the font awesome classes to a valid name to use in this component. (Why and How: README)
- Updated the README with the new features

### CODE REVIEW

**Comment by leandrosimoes** [comment]
@jgcmarins I just fix a missing var. Sorry about that.

**Comment by connoremma** [comment]
+1

**Comment by rgordon95** [comment]
Any chance this can be merged?

File: FontAwesomeIcons.js [hunk 1]
```diff
@@ -5,817 +5,1197 @@
...
+  yelp: "\uf1e9",
+  yenSign: "\uf157",
+  yinYang: "\uf6ad",
+  yoast: "\uf2b1",
+  youtube: "\uf167",
+  youtubeSquare: "\uf431",
+  zhihu: "\uf63f",
 }
 
-Object.keys(Icons).forEach((iconKey) => {
```
Comment (jgcmarins) [inline FontAwesomeIcons.js]: @leandrosimoes why removing this?

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (jgcmarins) [inline .vscode/launch.json]: can we remove this file?
add `.vscode` to `.gitignore`

Comment (leandrosimoes) [reply .vscode/launch.json]: Sure, I just put this file for run the tests with F5.

Comment (leandrosimoes) [reply .vscode/launch.json]: I just removed the file and sent another commit. Tks.

**Comment by KalebMatthews** [comment]
Anyway you can make a new build version so this is available on npm?

**Comment by KalebMatthews** [comment]
Thanks!

**Comment by jgcmarins** [comment]
done on npm: https://twitter.com/jgcmarins/status/1063124288507703298

**Pull request merged**